quickstarts/jfrog-platform/config.yml

```diff
@@ -0,0 +1,62 @@
+# quickstarts/example-category/example-quickstart/config.yml
+
+# This is automatically created by the system, this field should not be included by the author when creating a new quickstart.
+id:
+
+slug: jfrog-platform
+
+# Used as the display name for the quickstart
+title: JFrog Platform
+
+description: |
+  ## Why monitor JFrog Platform
+
+  The JFrog Platform is a leading universal end-to-end DevOps platform for automating, managing, securing, distributing, orchestrating, monitoring, and optimizing your Cl/ CD pipeline. These different elements of the JFrog platform need to be monitored in order for the enterprise to know in real-time about the health, performance and security of the DevOps supply chain.
+
+  ### JFrog Platform quickstart highlights
+
+  The integration will surface Platform Logs, Xray Violations data and system performance data available through Open Metrics, as pre-built dashboards within a JFrog Platform application made available within the New Relic One Programmable Observability platform.
+
+  - Monitor the JFrog platform in your familiar and convenient New Relic One Programmable Observability platform
+  - Pre-built JFrog dashboards depicting critical logs and metrics from the JFrog platform
+  - Utilizes FluentD, an open source and one of the most lightweight log processors and forwarders in the market
+
+summary: |
+  FluentD-based custom log forwarder for the JFrog platform for customers using New Relic's observability platform
+
+# Possible values: New Relic | Verified | Community
+# Please consult with pull request reviewers if you think your quickstart should have a support level other than "Community"
+level: Verified
+
+authors:
+  - JFrog
+  - Mahitha Byreddy
+
+keywords:
+  - newrelic
+  - newrelic partner
+  - apm
+  - open source monitoring
+
+# Relevant documentation for this quickstart
+documentation:
+  - name: JFrog Observability Solution
+    url: https://github.com/jfrog/log-analytics-newrelic
+    description: |
+      JFrog Github documentation
+
+
+# An icon or logo for the quickstart
+icon: icon.svg
```

Comment:
It looks like you're missing this `icon.svg` file. Could you add it into the pull request?

Reply:
added icon.svg